layouts/shortcodes/downloads-block.html

```diff
@@ -0,0 +1,301 @@
+<div class="card-wrapper">
+    <!--    SkyWalking APM-->
+    <div class="card item">
+        <div class="card-body">
+            <div class="title-box">
+                <span class="icon">S</span>
+                <h5 class="card-title">SkyWalking APM</h5>
+            </div>
+            <p class="card-text">
+                SkyWalking is an Observability Analysis Platform and Application Performance Management system.
+            </p>
+            <div class="doc-box">
+                <div class="flex-center">
+                    <img src="/images/feather.png" alt="" width="14px">
+                    <div class="btn-group">
+                        <button type="button" class="btn btn-sm dropdown-toggle" data-toggle="dropdown"
+                                aria-haspopup="true" aria-expanded="false">
+                            Source
+                        </button>
+                        <div class="dropdown-menu">
+                            <h6 class="dropdown-header">v8.2.0<span> | Oct. 27th, 2020</span></h6>
+                            <div class="dropdown-item" href="#">
+                                [<a href="https://www.apache.org/dyn/closer.cgi/skywalking/8.2.0/apache-skywalking-apm-8.2.0-src.tgz">src]</a>
+                                [<a href="https://downloads.apache.org/skywalking/8.2.0/apache-skywalking-apm-8.2.0-src.tgz.asc">asc]</a>
+                                [<a href="https://downloads.apache.org/skywalking/8.2.0/apache-skywalking-apm-8.2.0-src.tgz.sha512">sha512]</a>
```

Comment:
Why choose to make `src]` clickable instead of just `src` or `[src]`?

Reply:
And, do we need to manually update this html file when updating a new release?

Reply:
The HTML file is then changed to a template

Reply:
> Why choose to make `src]` clickable instead of just `src` or `[src]`?

Sorry, I copied the link from the old page and did not check it carefully